controllers/default.py

```diff
@@ -175,6 +177,16 @@ def index():
             % (request.application, request.application)
         )
     else:
+        # At this point the user has logged in
+        # add a jwt cookie for compatibility with bookserver
+        token = _create_access_token(
+            {"sub": auth.user.username}, expires=datetime.timedelta(hours=24 * 30)
+        )
+        # set cookie
+        response.cookies["access_token"] = token
+        response.cookies["access_token"]["expires"] = 24 * 3600 * 90
```

Comment:
Why are the expires times different here (90 days, vs 30 days a few lines ago)? Does it matter?

Reply:
We need to think about the expiration stuff.  The fastapi_login plugin respects what is in the encrypted token.  So that is one level of expiration.  You are right that it makes no sense to have the cookie expiration longer than the token expiration.    But we definitely will want to have tokens lasting more than 12 hours.